src/TAlias.cpp

```diff
@@ -221,19 +222,19 @@ bool TAlias::match(const QString& toMatch)
         }
     }
 
-END:
-    {
-        TLuaInterpreter * pL = mpHost->getLuaInterpreter();
-        pL->setCaptureGroups( captureList, posList );
-        // call lua trigger function with number of matches and matches itselves as arguments
-        execute();
-        pL->clearCaptureGroups();
-    }
+END : {
```

Comment:
I think this layout is wrong - as the closing brace is out of position on left margin.  Perhaps the label should be on a separate line to the code block opening brace?